CHANGELOG.md

```diff
@@ -8,13 +8,17 @@
   * Add new endpoint `DELETE /api/v1/cart/deliveries/items` to be able to remove all cart items from all deliveries but keeping delivery info and other cart data untouched
 * Add new method `SumShippingGrossWithDiscounts` to the cart domain which returns gross shipping costs for the cart 
 * When using the `ItemSplitter` to split items in items with single qty (`SplitInSingleQtyItems`) the split discounts are reversed to make splitting the row total stable.
+* **Breaking**: Delivery discount sum calculations `SumNonItemRelatedDiscountAmount`,`SumItemRelatedDiscountAmount` now take discount on shipping costs into account
```

Comment:
`SumTotalTaxAmount` breaking as well now also with shipping

Reply:
`SumTotalDiscountAmount` breaking since also shipping discounts involved

Reply:
`shippingItem.PriceGross()` can be added as new, currently only in the GraphQL changelog